internal/pkg/container/config.go

```diff
@@ -27,3 +27,9 @@ func ImageParentDir() string {
 func ImageFile(name string) string {
 	return path.Join(ImageParentDir(), name+".img")
 }
+
+// InitramfsBootPath returns the dracut built initramfs path, as dracut built initramfs inside container
+// the function returns host path of the built file
+func InitramfsBootPath(image, kver string) string {
+	return path.Join(RootFsDir(image), "boot", "initramfs-"+kver+".img")
```

Comment:
I'm not certain how I feel about explicitly only detecting an initramfs that matches the kver to be booted. I want to see if we can overlay a kernel override onto the dracut image and boot that successfully. If so, then we should be able to boot any arbitrary initramfs and not worry about the kernel matching; but if not then it probably makes sense to be strict about it as you have here.